cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/dataset/MetadataDatasetTest.java

```diff
@@ -1137,42 +1137,34 @@ public void apply() throws Exception {
         searchResults = dataset.search(namespaceId, "*", targets, nameAsc, 0, 2, 0, null, false,
                                        EnumSet.allOf(EntityScope.class));
         Assert.assertEquals(ImmutableList.of(flowEntry, dsEntry), searchResults.getResults());
-        Assert.assertEquals(ImmutableList.of(flowEntry, dsEntry, appEntry), searchResults.getAllResults());
```

Comment:
If we do that, then we reintroduce the bug that was fixed in the last PR: https://issues.cask.co/browse/CDAP-7930.